docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -1,26 +1,51 @@
 ////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= YAML Configuration
+// meta:assemblies/assembly_configuring-your-quarkus-applications-using-a-yaml-file.adoc
+[id="config-yaml"]
+= Configure Quarkus applications by using a YAML file
 include::_attributes.adoc[]
+:diataxis-type: howto
 :categories: core
-:summary: YAML as a Configuration Source.
+:summary: Use `application.yaml` instead of `application.properties`.
+:context: config-yaml-howto
 
-https://en.wikipedia.org/wiki/YAML[YAML] is a very popular format. Kubernetes relies heavily on the YAML format to
-write the various resource descriptors.
+Optionally, you can configure properties for your application with a YAML-formatted file, `application.yaml`, instead of the default `application.properties` file.
 
-Quarkus offers the possibility to use YAML in addition to the standard Java Properties file.
+// meta:include::modules/con_overview-quarkus-configuration.adoc[leveloffset=+1]
+[id="con_overview_{context}"]
+== Overview
 
-== Enabling YAML Configuration
+To configure properties in `application.yaml`, you define nested elements by using indentation and key-value pairs.
 
-To enable YAML configuration, add the `quarkus-config-yaml` extension:
+To make writing `application.yaml` easier, you can and paste snippets from some YAML configuration examples into your `application.yaml` file.
```

Comment:
What's the purpose of this sentence?

I don't see how it will help writing `application.yaml` files? Especially since we don't provide any snippets in our guides.